bugbug/bug_features.py

```diff
@@ -333,6 +345,7 @@ def transform(self, bugs):
         results = []
 
         reporter_experience_map = defaultdict(int)
+        author_ids = get_author_ids()
```

Comment:
Only call get_author_ids if commit_data is not None (like we do for `self.commit_map`)

Reply:
(this will probably fix CI too)

Reply:
i dont get it which condition to put here? error saying no author_email in commits.

Reply:
the same we have a few lines earlier for `self.commit_map`.

Reply:
Done